### ISSUE TITLE
Plugin will never initialize on some devices due to CMP

### ISSUE THREAD

**Issue by ThomasEcalle** [AppLovin/AppLovin-MAX-Flutter#218]
### MAX Plugin Version

12.5.0

### Flutter Version

3.10.0

### Device/Platform Info

Android 13, Samsung S20

### Current Behavior

On some devices (not all), as a Samsung S20 with Android 13, the AppLovin SDK never finish initialization and get stuck forever. No exception thrown.

This is only happening when trying to show the Google CMP dialog.
If we try to show it, it works on certain devices and loop forever with others.

If we don't show it, plugin initializes itself correctly.

### Expected Behavior

Here is how the pakcage is initialized:

```
try {
    debugPrint('Initializing Applovin MAX SDK in main with key: $applovinMaxKey');
    AppLovinMAX.setConsentFlowDebugUserGeography(ConsentFlowUserGeography.gdpr);
    AppLovinMAX.setTermsAndPrivacyPolicyFlowEnabled(true);
    AppLovinMAX.setPrivacyPolicyUrl('https://www.gamby.fr/privacy');
    AppLovinMAX.setTermsOfServiceUrl('https://www.gamby.fr/terms');
    await AppLovinMAX.initialize(applovinMaxKey);
    AppLovinMAX.setVerboseLogging(true);
    AppLovinMAX.showMediationDebugger();
    debugPrint('Applovin MAX SDK initialized in main');
  } on PlatformException catch (plateformException) {
    debugPrint('ONEError initializing Applovin MAX SDK: $plateformException');
  } catch (error) {
    debugPrint('TWOError initializing Applovin MAX SDK: $error');
  }
```

The expected behavior should be to be able to print `Applovin MAX SDK initialized in main` after the package initialization.

### How to Reproduce

Actually it works on some devices and it does not work on others but I have no clue why.
But, on my Samsung S20 with Android 13 it NEVER works with this code:

```
try {
    debugPrint('Initializing Applovin MAX SDK in main with key: $applovinMaxKey');
    AppLovinMAX.setConsentFlowDebugUserGeography(ConsentFlowUserGeography.gdpr);
    AppLovinMAX.setTermsAndPrivacyPolicyFlowEnabled(true);
    AppLovinMAX.setPrivacyPolicyUrl('https://www.gamby.fr/privacy');
    AppLovinMAX.setTermsOfServiceUrl('https://www.gamby.fr/terms');
    await AppLovinMAX.initialize(applovinMaxKey);
    AppLovinMAX.setVerboseLogging(true);
    AppLovinMAX.showMediationDebugger();
    debugPrint('Applovin MAX SDK initialized in main');
  } on PlatformException catch (plateformException) {
    debugPrint('ONEError initializing Applovin MAX SDK: $plateformException');
  } catch (error) {
    debugPrint('TWOError initializing Applovin MAX SDK: $error');
  }
```

**Comment by alhiwatan** [comment]
@ThomasEcalle. thanks for raising the issue, but I was not able to reproduce it with the devices I have.  Could you try to see this on a simple Google example app with your problematic device?   Then, we will know more about where the issue is.  You can pick one of the examples that Google provides here.

https://github.com/googleads/googleads-mobile-android-examples

I tried BannerExample in the Java/Admob, and the dialog showed up when it started.

**Comment by alhiwatan** [comment]
@ThomasEcalle, thanks for raising the issue.   This might be related to #212.  Could you check that the initialization is somehow called more than once?

**Comment by alhiwatan** [comment]
@ThomasEcalle, we have just released [3.10.1](https://github.com/AppLovin/AppLovin-MAX-Flutter/releases/tag/release_3_10_1). Please feel free to reopen if you still have the issue.
